src/main/scala/io/sdkman/changelogs/package.scala

```diff
@@ -44,4 +47,9 @@ package object changelogs {
 
   def updateCandidateDefault(candidate: String, version: String)(implicit db: MongoDatabase): Document =
     db.getCollection("candidates").findOneAndUpdate(Filters.eq("candidate", candidate), Updates.set("default", version))
+
+  def updateVersion(version: String, newVersion: String)(implicit db: MongoDatabase): Unit = {
+    db.getCollection("versions").updateMany(Filters.eq("version", version), Updates.set("version", newVersion))
```

Comment:
`updateMany` could potentially be dangerous as version could be very ambiguous. I like the idea of adding this helper function though...

Reply:
Ok so i think a better way would be to find them one by one and after that update the version?